components/Python/Scikit-learn/sink/s3_file_sink/main.py

```diff
@@ -0,0 +1,50 @@
+from __future__ import print_function
+
+import argparse
+import os
+import boto3
+import uuid
+import time
+
+from parallelm.components import ConnectableComponent
+from parallelm.ml_engine.python_engine import PythonEngine
+from parallelm.mlops import mlops as mlops
+
+
+class S3FileSink(ConnectableComponent):
+
+    def __init__(self, engine):
+        super(self.__class__, self).__init__(engine)
+
+    def _materialize(self, parent_data_objs, user_data):
+        file_path = str(parent_data_objs[0])
+        self._save_file(file_path)
+        return []
+
+    def _save_file(self, file_path):
+
+        # Initialize mlops
+        mlops.init()
```

Comment:
do each component have to init mlops?

Reply:
At least one component in the pipeline needs to perform the "mlops.init()" 
given these are connectable components not clear what other components are (or aren't) performing the init.

however calling into mlops.init() by each of the component isn't an issue, from the usage prospective, not clear on the resource concerns if any on the server.. (experts can comment)